music21/stream/iterator.py

```diff
@@ -337,6 +351,9 @@ def __getitem__(self, k):
 
         (nothing is printed)
         '''
+        if isinstance(k, type):
+            self.addFilter(k)
+            return self
```

Comment:
This is here to support chained lookups like s[x][y]... needs docs and test if we want it.

Reply:
Ah -- I don't think that this will exactly work.  maybe remove it for now and then we can put it back in during a refactor of Stream.getItem, because otherwise we'll end up doing too much duplicated code to get something like: Stream.parts['Note'][4] to work, and it'd be weird because `.parts` is a normal, non recursive iterator, but we're now training people to think of `Stream[Note]` as making a recursive iterator.